CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Guidelines for contribution of new pipelines
+
+> [!TIP]
+> The process of submitting a pipeline for inclusion is done entirely through the review of
+> a pull request. Most of the pre-review checklist is codified in the "New pipeline" issue
+> template. Submissions of new pipelines that do not use this template will not be merged.
+
+## Pre-review checklist
+
+### General information about the pipeline
+
+- [ ] Descriptive title for the pipeline
+- [ ] Authorship information with appropriate CRediT roles
+- [ ] Abstract 
+
+### Code checklist
```

Comment:
I was thinking such a checklist could go in the top of the PR template (maybe link to it to avoid duplication?):

Before submitting this pull request, I have:

- [ ] Formatted my code: with appropriate indentation, clear variable names, clear function names, comments where needed, etc.
- [ ] Tested my code with the examples
- [ ] Tested my code in various contexts (region, species, time, etc.)
- [ ] Clearly documented the script yml files and in the pipeline metadata tab.
- [ ] Checked that the GitHub actions pass
- [ ] ...